src/zenml/pipelines/pipeline_definition.py

```diff
@@ -882,9 +882,12 @@ def _run(
                 if run:
                     run_url = dashboard_utils.get_run_url(run)
                     if run_url:
-                        logger.info(
-                            f"Dashboard URL for Pipeline Run: {run_url}"
-                        )
+                        # Display the dashboard URL directly without panel to avoid wrapping
+                        from zenml.console import console
+                        
+                        console.print()
+                        console.print(f"Dashboard URL: [bright_blue underline]{run_url}[/bright_blue underline]", overflow="ignore", crop=False)
+                        console.print()
```

Comment:
```suggestion
                        logger.info(
                            f"Dashboard URL for Pipeline Run: {run_url}"
                        )
```

I would revert back to the logger here. We do not use the console outside of the CLI.